src/date.ts

```diff
@@ -15,6 +15,14 @@ export function isFuture(date: Date | string | number): boolean {
   return isAfter(date, new Date());
 }
 
+export function isValidGermanDate(date: string): boolean {
+  return /^(\d{1,2}[.]\d{1,2}[.]\d{4})$/.test(date);
+}
+
+export function isValidTime(time: string): boolean {
```

Comment:
Actually I think the name for this is not specific enough. How about `isHoursAndMinutes`?

Reply:
How about "isTimeWithHoursAndMinutes"?

Reply:
Sounds good.